### PR TITLE
Capture video width/height

### PR DESCRIPTION
**Pull request by erikpena**

### CODE REVIEW

File: src/mux-analytics.brs [hunk 1]
```diff
@@ -60,6 +60,7 @@ function runBeaconLoop()
     m.top.video.ObserveField("state", m.messagePort)
     m.top.video.ObserveField("content", m.messagePort)
     m.top.video.ObserveField("control", m.messagePort)
+    m.top.video.ObserveField("streamingSegment", m.messagePort)
```
Comment (jsanford8) [inline src/mux-analytics.brs:63]: This is backwards-compatible, right? As in if the firmware version doesn't support this, this essentially no-ops?

**Review by jsanford8 — COMMENTED** [review]
one small comment about backwards-compatibility, but this looks great otherwise

File: src/mux-analytics.brs (hunk 1 above)
Comment (nbirkenshaw-mux) [reply src/mux-analytics.brs:63]: This appears to be the case. observeField for non-existent fields seems to do nothing - setField on the same non-existent field does blow up.

**Review by nbirkenshaw-mux — APPROVED** [review]

**Comment by jsanford8** [comment]
Closing in favor of #62

**Pull request closed without merging**